src/components/Company/Offers/Manage/CompanyOffersManagementWidget.spec.js

```diff
@@ -75,13 +91,15 @@ describe("App", () => {
 
         await waitFor(() => {
             expect(screen.getByText("Offers Management")).toBeInTheDocument();
+        }, {
+            timeout: 2000,
         });
     });
 
     test("Error fetching offers", async () => {
-        companyOffersService.fetchCompanyOffers = jest.fn(() => new Promise(() => {
-            throw "Error fetching offers";
-        }));
+        companyOffersService.fetchCompanyOffers.mockImplementationOnce(() => new Promise((resolve, reject) => setTimeout(() => {
+            reject("Error fetching offers");
+        }, 1000)));
```

Comment:
Why not reject immediately? These types of mocks (with the set timeout) make the test run much longer, and unnecessarily in my opinion